# HG changeset patch
# User Rik <rik@octave.org>
# Date 1346769946 25200
#      Tue Sep 04 07:45:46 2012 -0700
# Node ID 03ef3a183129380da205e92025b05cdec5ec337c
# Parent  2dd4f5930108d1340101813916d98e4c4a7ddc39
build: Use INC rather than INCLUDES for consistent naming in libinterp Makefile.am.

* libinterp/Makefile.am, libinterp/interp-core/module.mk,
libinterp/interpfcn/module.mk, libinterp/octave-value/module.mk,
libinterp/operators/module.mk, libinterp/parse-tree/module.mk:
Use INC rather than INCLUDES for consistent naming.

diff --git a/libinterp/Makefile.am b/libinterp/Makefile.am
--- a/libinterp/Makefile.am
+++ b/libinterp/Makefile.am
@@ -92,22 +92,21 @@ EXTRA_DIST = \
   version.in.h \
   $(BUILT_DISTFILES)
 
 octinclude_HEADERS = \
   interpfcn/graphics-props.cc \
   parse-tree/oct-gperf.h \
   builtins.h \
   octave.h \
-  $(OV_INCLUDES) \
-  $(OV_SPARSE_INCLUDES) \
-  $(PT_INCLUDES) \
-	$(OPERATOR_INCLUDES) \
-  $(INTERP_CORE_INCLUDES) \
-  $(INTERPFCN_INCLUDES)
+  $(OCTAVE_VALUE_INC) \
+  $(PARSE_TREE_INC) \
+  $(OPERATORS_INC) \
+  $(INTERP_CORE_INC) \
+  $(INTERPFCN_INC)
 
 nodist_octinclude_HEADERS = \
   interp-core/mxarray.h \
   interpfcn/defaults.h \
   interpfcn/graphics.h \
   oct-conf.h \
   version.h
 
diff --git a/libinterp/interp-core/module.mk b/libinterp/interp-core/module.mk
--- a/libinterp/interp-core/module.mk
+++ b/libinterp/interp-core/module.mk
@@ -1,21 +1,21 @@
 EXTRA_DIST += \
   interp-core/module.mk \
   interp-core/gl2ps.c \
   interp-core/mxarray.in.h \
   interp-core/oct-errno.in.cc
 
-JIT_INCLUDES = \
+JIT_INC = \
   interp-core/jit-util.h \
   interp-core/jit-typeinfo.h \
   interp-core/jit-ir.h \
   interp-core/pt-jit.h
 
-INTERP_CORE_INCLUDES = \
+INTERP_CORE_INC = \
   interp-core/Cell.h \
   interp-core/c-file-ptr-stream.h \
   interp-core/comment-list.h \
   interp-core/cutils.h \
   interp-core/defun-dld.h \
   interp-core/defun-int.h \
   interp-core/display.h \
   interp-core/dynamic-ld.h \
@@ -52,17 +52,17 @@ INTERP_CORE_INCLUDES = \
   interp-core/sparse-xpow.h \
   interp-core/txt-eng-ft.h \
   interp-core/txt-eng.h \
   interp-core/unwind-prot.h \
   interp-core/xdiv.h \
   interp-core/xnorm.h \
   interp-core/xpow.h \
   interp-core/zfstream.h \
-  $(JIT_INCLUDES)
+  $(JIT_INC)
 
 JIT_SRC = \
   interp-core/jit-util.cc \
   interp-core/jit-typeinfo.cc \
   interp-core/jit-ir.cc \
   interp-core/pt-jit.cc
 
 C_INTERP_CORE_SRC = \
diff --git a/libinterp/interpfcn/module.mk b/libinterp/interpfcn/module.mk
--- a/libinterp/interpfcn/module.mk
+++ b/libinterp/interpfcn/module.mk
@@ -1,14 +1,14 @@
 EXTRA_DIST += \
   interpfcn/module.mk \
   interpfcn/defaults.in.h \
   interpfcn/graphics.in.h
 
-INTERPFCN_INCLUDES = \
+INTERPFCN_INC = \
   interpfcn/data.h \
   interpfcn/debug.h \
   interpfcn/defun.h \
   interpfcn/dirfns.h \
   interpfcn/error.h \
   interpfcn/file-io.h \
   interpfcn/help.h \
   interpfcn/input.h \
diff --git a/libinterp/octave-value/module.mk b/libinterp/octave-value/module.mk
--- a/libinterp/octave-value/module.mk
+++ b/libinterp/octave-value/module.mk
@@ -1,32 +1,32 @@
 EXTRA_DIST += \
   octave-value/module.mk
 
-OV_INTTYPE_INCLUDES = \
+OV_INTTYPE_INC = \
   octave-value/ov-base-int.h \
   octave-value/ov-base-int.cc \
   octave-value/ov-int-traits.h \
   octave-value/ov-int16.h \
   octave-value/ov-int32.h \
   octave-value/ov-int64.h \
   octave-value/ov-int8.h \
   octave-value/ov-intx.h \
   octave-value/ov-uint16.h \
   octave-value/ov-uint32.h \
   octave-value/ov-uint64.h \
   octave-value/ov-uint8.h
 
-OV_SPARSE_INCLUDES = \
+OV_SPARSE_INC = \
   octave-value/ov-base-sparse.h \
   octave-value/ov-bool-sparse.h \
   octave-value/ov-cx-sparse.h \
   octave-value/ov-re-sparse.h
 
-OV_INCLUDES = \
+OCTAVE_VALUE_INC = \
   octave-value/ov-base-diag.h \
   octave-value/ov-base-diag.cc \
   octave-value/ov-base-mat.h \
   octave-value/ov-base-mat.cc \
   octave-value/ov-base-scalar.h \
   octave-value/ov-base-scalar.cc \
   octave-value/ov-base.h \
   octave-value/ov-bool-mat.h \
@@ -61,17 +61,18 @@ OV_INCLUDES = \
   octave-value/ov-re-mat.h \
   octave-value/ov-scalar.h \
   octave-value/ov-str-mat.h \
   octave-value/ov-struct.h \
   octave-value/ov-type-conv.h \
   octave-value/ov-typeinfo.h \
   octave-value/ov-usr-fcn.h \
   octave-value/ov.h \
-  $(OV_INTTYPE_INCLUDES)
+  $(OV_INTTYPE_INC) \
+  $(OV_SPARSE_INC)
 
 OV_INTTYPE_SRC = \
   octave-value/ov-int16.cc \
   octave-value/ov-int32.cc \
   octave-value/ov-int64.cc \
   octave-value/ov-int8.cc \
   octave-value/ov-uint16.cc \
   octave-value/ov-uint32.cc \
diff --git a/libinterp/operators/module.mk b/libinterp/operators/module.mk
--- a/libinterp/operators/module.mk
+++ b/libinterp/operators/module.mk
@@ -125,17 +125,17 @@ OPERATORS_SRC = \
   operators/op-str-str.cc \
   operators/op-struct.cc \
   operators/op-ui16-ui16.cc \
   operators/op-ui32-ui32.cc \
   operators/op-ui64-ui64.cc \
   operators/op-ui8-ui8.cc
 
 ## These look like included header files to Autotools build process
-OPERATOR_INCLUDES = \
+OPERATORS_INC = \
   operators/op-dm-template.cc \
   operators/op-dms-template.cc \
   operators/op-int.h \
   operators/op-pm-template.cc \
   operators/ops.h
 
 ## Special rules for sources which must be built before rest of compilation.
 operators/ops.cc: $(OPERATORS_SRC) mkops
diff --git a/libinterp/parse-tree/module.mk b/libinterp/parse-tree/module.mk
--- a/libinterp/parse-tree/module.mk
+++ b/libinterp/parse-tree/module.mk
@@ -1,25 +1,25 @@
 EXTRA_DIST += \
   parse-tree/module.mk \
   parse-tree/octave.gperf
 
-PARSER_INCLUDES = \
+PARSER_INC = \
   parse-tree/lex.h \
   parse-tree/parse.h \
   parse-tree/parse-private.h
 
 PARSER_SRC = \
   parse-tree/lex.ll \
   parse-tree/oct-parse.yy
 
 lex.lo lex.o oct-parse.lo oct-parse.o: \
   AM_CXXFLAGS := $(filter-out -Wold-style-cast, $(AM_CXXFLAGS))
 
-PT_INCLUDES = \
+PARSE_TREE_INC = \
   parse-tree/pt-all.h \
   parse-tree/pt-arg-list.h \
   parse-tree/pt-assign.h \
   parse-tree/pt-binop.h \
   parse-tree/pt-bp.h \
   parse-tree/pt-cbinop.h \
   parse-tree/pt-cell.h \
   parse-tree/pt-check.h \
@@ -39,17 +39,17 @@ PT_INCLUDES = \
   parse-tree/pt-misc.h \
   parse-tree/pt-pr-code.h \
   parse-tree/pt-select.h \
   parse-tree/pt-stmt.h \
   parse-tree/pt-unop.h \
   parse-tree/pt-walk.h \
   parse-tree/pt.h \
   parse-tree/token.h \
-  $(PARSER_INCLUDES)
+  $(PARSER_INC)
 
 PARSE_TREE_SRC = \
   parse-tree/pt-arg-list.cc \
   parse-tree/pt-assign.cc \
   parse-tree/pt-binop.cc \
   parse-tree/pt-bp.cc \
   parse-tree/pt-cbinop.cc \
   parse-tree/pt-cell.cc \
